server/database/banano/known-accounts/gambling.json

```diff
@@ -63,5 +63,15 @@
         "address": "ban_1eub1ezmefs1im56oomprzddwhp5s9c6brhx755ftjfjz45uytputarmcwto",
         "alias": "Banano Sports Pools MLB playoffs and teams",
         "owner": "Elementalennui"
+    },
```

Comment:
Thanks for the PR 👍 
Could you take a look at `ban_3nuq9s1ot8dzk4i1w1zxejnk6qgbr7zgaw9rxagokbwdxw96z4fbk85girf6` & decide which category it best belongs in - gambling or faucet?